src/reference/informative/informC02_units_reference.rst

```diff
@@ -8,4 +8,300 @@
+        ╵╵└╴├─ units: smidgen
+        ╵└╴╴├─ units: dash
+        └╴╴╴├─ units: spoonful
+            └─ units: dollop   
+
+
+    .. container:: toggle
+
+      .. container:: header
+
+        See CellML syntax
+
+      .. code-block:: xml
+
+        <!-- Inside the file "crust_recipes.cellml": -->
+        <model name="PieCrustRecipes">
+
+          <component name="HazelnutLavenderCrust">
+
+            <!-- These units are built-in so do not change. -->
+            <variables name="ground_hazelnut" units="gram" />
+            <variables name="egg" units="dimensionless" />
+            <variables name="flour" units="gram" />
+            <variables name="sugar" units="gram" />
+
+            <!-- These units are defined for this, their local scope, below. -->
+            <variables name="water" units="spoonful" />
+            <variables name="salt" units="dash" />
+            <variables name="lavender_flowers" units="smidgen" />
+            ...
+          </component>
+
+          <component name="CheeseAndAlmondCrust">
+            ...
+          </component>
+
+          <!-- Local units definitions for spoonful, dash, and smidgen. -->
+          <units name="spoonful">
+            <unit units="litre" prefix="milli" multiplier="15" />
+          </units>
+          <units name="dash">
+            <unit units="gram" multiplier="5" />
+          </units>
+          <units name="smidgen">
+            <unit units="gram" multiplier="1" />
+          </units>
+          <units name="dollop">
+            <unit units="litre" prefix="milli" multiplier="20" />
+          </units>
+        </model>
+
+        <!-- Inside the file "filling_recipes.cellml": -->
+        <model name="PieFillingRecipes">
+
+          <component name="BlueberryCinnamonFilling">
+            <!-- These units are built-in, so do not change.  -->
+            <variables name="blueberries" units="gram" />
+            <variables name="sugar" units="dimensionless" />
+            <variables name="cornflour" units="gram" />
+
+            <!-- These units are defined for use in this, their local scope, below. -->
+            <variables name="cinnamon" units="smidgen" />
+            <variables name="water" units="spoonful" />
+
+            <math>
+                ...
+            </math>
+          </component>
+
+          <component name="AppleAndPearFilling">
+            ...
+          </component>
+
+          <!-- Local units definitions for spoonful and smidgen. -->
+          <units name="spoonful">
+            <unit units="litre" prefix="milli" multiplier="5" />
+          </units>
+          <units name="smidgen">
+            <unit units="gram" multiplier="20" />
+          </units>
+
+        </model>
+
+    This is where the idea of *scope* becomes important.  
+    As it stands, there is no conflict between the two different definitions of :code:`spoonful` and :code:`dash`, because each of the components refers to *its own definition* of these units.
```

Comment:
I believe you mean `smidgen` rather than `dash`? (`dash` is only defined in the `PieCrustRecipes` model.

Reply:
Nice catch!  c8fb09d